storage/object/api-cli/migrating-bucket-policy-version.mdx

```diff
@@ -0,0 +1,133 @@
+---
+meta:
+  title: Migrating from a bucket policy version to another
+  description: This page shows how to migrate from a bucket policy version to a more recent one.
+content:
+  h1: Migrating from a bucket policy version to another
+  paragraph: This page shows how to migrate from a bucket policy version to a more recent one.
+tags: object storage object-storage bucket-policy version migrate upgrade
+dates:
+  validation: 2023-10-18
+  posted: 2023-10-02
+categories:
+  - storage
+  - object-storage
+---
+
+Several bucket policy versions coexist in Scaleway Object Storage, and each version has its own mode of operation. A more recent version can help manage the access rights to an existing bucket.
+
+Refer to the [bucket policy versions](/storage/object/api-cli/bucket-policy/#bucket-policy-versions) documentation for more information.
+
+## Retrieving a bucket's policy
+
+1. Open a terminal and run the following command to save the policy of a bucket in a `migrate-policy.json` file:
+    ```
+    aws s3api get-bucket-policy --bucket <BUCKET_NAME> --query Policy --output text | jq > migrate-policy.json
+    ```
+2. Open the `migrate-policy.json` file in a code editor. Your current bucket policy displays.
+
+## Migrating from V2 to V3
+
+1. Remove any `Deny` statements from your V2 bucket policy, as they do not have any effects.
```

Comment:
Here and everywhere, I don't think we should talk about v2 and V3 since this is the way we refered to them internally